### PR TITLE
Adding 'Detecting Deep Moist Convection' to openeo-processes-dask

### PR DESCRIPTION
**Pull request by koenifra**
Adding the process and test file for Detecting Deep Moist Convection (ddmc.py and test_ddmc.py) to openeo-processes-dask.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ rasterio = { version = "^1.3.4", optional = true }
 dask-geopandas = { version = ">=0.2.0,<1", optional = true }
 xgboost = { version = ">=1.5.1", optional = true }
 rioxarray = { version = ">=0.12.0,<1", optional = true }
-openeo-pg-parser-networkx = { version = ">=2023.5.1", optional = true }
+openeo-pg-parser-networkx = "2024.7"
```
Comment (ValentinaHutter) [inline pyproject.toml:35]: The current version of openeo-processes-dask needs the openeo-pg-parser-networkx to be 2024.7, but versions above that one should be allowed, too. 
You could update the file with 
```
openeo-pg-parser-networkx = { version = ">=2024.7", optional = true }
```

**Review by ValentinaHutter — COMMENTED** [review]
Looks good to me! 

You included a file called example%20file.geoparquet - is this required?

File: pyproject.toml (hunk 1 above)
Comment (koenifra) [reply pyproject.toml:35]: I don't think the example%20file.geoparquet is required.

Comment (ValentinaHutter) [reply pyproject.toml]: okay - then please delete the file within your nextcommit :)

**Comment by koenifra** [comment]
All requested changes are committed.

**Pull request merged**